magda-web-client/src/Components/Dataset/DatasetPage.tsx

```diff
@@ -122,13 +125,37 @@ const DatasetPage: FunctionComponent<PropsType> = (props) => {
                                 />
                             </div>
                         ) : null}
+                        {dataset.informationSecurity?.classification && (
+                            <SecClassification
+                                secClass={
+                                    dataset.informationSecurity?.classification
+                                }
+                            />
+                        )}
+                        {dataset.informationSecurity?.disseminationLimits && (
+                            <Sensitivity
+                                sensitivityList={
+                                    dataset.informationSecurity
+                                        .disseminationLimits
+                                }
+                            />
+                        )}
                         {dataset.contactPoint ? (
                             <ContactPoint contactPoint={dataset.contactPoint} />
                         ) : (
                             <></>
                         )}
-                        <TagsBox tags={dataset.tags} />
-
+                        {dataset.currencyStatus ? (
+                            <div>
+                                Currency of the dataset:{" "}
```

Comment:
Personal preference maybe, but I think this'd be neater if it just said "Currency:"

Reply:
Fair enough. I thought that for the uninitiated, it might appear to be the money-related currency.